lstchain/tools/lstchain_create_calibration_file.py

```diff
@@ -223,15 +224,24 @@ def start(self):
                 self.processor.calculate_calibration_coefficients(event)
 
                 # write calib and pixel status
-                self.log.debug("Write pixel_status data")
+                self.log.info("Write pixel_status data")
                 self.writer.write('pixel_status', status_data)
 
-                self.log.debug("Write calibration data")
+                self.log.info("Write calibration data")
                 self.writer.write('calibration', calib_data)
+                self.n_calib += 1
+
                 if self.one_event:
                     break
 
     def finish(self):
+
+        self.log.info(f"Written {self.n_calib} calibration events")
+        if self.n_calib == 0:
+             self.log.warning(f"!!! No calibration events in the output file !!! : ")
```

Comment:
Wouldn't it make sense to exit with an error code in this case? I.e. even use `error` or `critical` and then do `self.exit(1)`?

Reply:
yes, indeed. Perhaps better "critical" (strictly speaking this is not an error)

Reply:
But I would not exit before the file is closed and provenance is written, better to keep track of this type of problems

Reply:
Provenance will still be written and files closed  (if self.enter_context is used to open the file / writer), even if you call exit, because writing provenance is in a "finally" block:

https://github.com/cta-observatory/ctapipe/blob/c44226a5ee6b28ea5cc3de6d82e828aedf319b30/ctapipe/core/tool.py#L448-L450